src/styles/partials/desktop/_header.scss

```
@@ -1,120 +1,232 @@
-html {
-  body {
-    header.aj-1j1s-header {
-      background-color: var(--white);
-
-      .aj-1j1s-header-logo-container {
-        border-bottom: 0.0625rem solid #e6e6e6;
-        padding: 0.625rem;
-
-        .container {
-          display: flex;
-          justify-content: center;
-          .aj-1j1s-header-right-links {
-            margin-left: auto;
-
-            .button {
-              margin-left: 20px;
-
-              &.is-mobile {
-                display: none;
-              }
-            }
-          }
+.aj-1j1s-header {
+  background-color: var(--white);
+
+  .active {
+    font-weight: 700;
+    text-decoration: underline !important;
+    text-underline-offset: 0.2rem;
+  }
+
+  a {
+    text-underline-offset: 0.2rem;
+    color: inherit;
+    text-decoration: none;
+  }
+
+  a:hover {
+    text-decoration: underline;
+  }
+
+  &__title {
+    font-size: 1.25rem;
+    font-weight: 700;
+    color: #000;
+    text-decoration: none;
+    font-family: Marianne, arial, sans-serif;
+    margin-top: 1rem;
+  }
+
+  &-modal {
+    display: none;
+  }
+
+  &-logo-container {
+    padding-block: 1rem;
+    padding-inline: 1rem;
+    padding-bottom: 1.65em;
+
+    .container {
+      display: flex;
+      justify-content: space-between;
+    }
+
+    &__col-left {
+      display: flex;
+      align-items: center;
+      gap: 4rem;
+      .aj-1j1s-responsive-menu-button {
+        display: none;
+      }
+    }
+
+    &__col-right {
+      margin-left: auto;
+
+      .button {
+        margin-left: 20px;
+
+        &.is-mobile {
+          display: none;
         }
+      }
+    }
+
+    img {
+      margin-top: 0.1rem;
+
+      &.rf {
+        width: 5.5rem;
+      }
 
-        img {
-          margin-right: 20px;
+      &.fr {
+        width: 11.0625rem;
+      }
+    }
+  }
 
-          &.rf {
-            width: 9.5rem;
-          }
+  &-nav {
+    border-top: 1px solid #e5e5e5;
+    .container {
+      max-width: 78rem !important;
+    }
 
-          &.fr {
-            width: 11.0625rem;
-          }
+    &__menu {
+      display: flex;
+      justify-content: space-between;
+      flex-direction: row;
+      align-items: center;
+      padding: 0;
+      margin: 0;
+      position: relative;
+      padding-left: 1.5rem;
+      padding-right: 0;
+      width: 100%;
+
+      &-submenus {
+        color: #161616;
+        z-index: 5;
+        position: absolute;
+        background-color: #fff;
+        border-left: 1px solid #e5e5e5;
+        border-right: 1px solid #e5e5e5;
+        border-bottom: 1px solid #e5e5e5;
+        border-radius: 0 0 20px 20px;
+        overflow: hidden;
+        margin-bottom: 1rem;
+        margin-left: -0.6rem;
+        margin-top: 0.57rem;
+        padding-inline-start: 0;
+
+        li {
+          border-top: 1px solid #e5e5e5;
+          width: 100%;
+          padding: 0.5rem;
+          display: block;
+          padding-block: 0.8rem;
+          padding-inline: 1rem;
         }
```

```suggestion
        li {
          border-top: 1px solid #e5e5e5;
          width: 100%;
        }

        a {
          padding: 0.5rem;
          display: block;
          padding-block: 0.8rem;
          padding-inline: 1rem;
        }
```

pour avoir le click sur tout l'élément

---

C'est juste